src/wallet/wallet.cpp

```diff
@@ -3266,6 +3267,14 @@ bool CWallet::CreateTransaction(interfaces::Chain::Lock& locked_chain, const std
 
                 // vouts to the payees
                 coin_selection_params.tx_noinputs_size = 11; // Static vsize overhead + outputs vsize. 4 nVersion, 4 nLocktime, 1 input count, 1 output count, 1 witness overhead (dummy, flag, stack size)
+
+                // Account for the fee output in the tx.
+                if (g_con_elementsmode && nSubtractFeeFromAmount == 0) {
```

Comment:
Hmm why condition on the value of nSubtractFeeFromAmount? That doesn't change whether there is a fee output taking up space, right?